user/jwt.md

```diff
@@ -0,0 +1,143 @@
+---
+title: JWT addon
+layout: en
+permalink: /user/jwt/
+---
+
+Integration between Travis-CI and third-party services like Sauce Labs relies
+on [encrypted variables](http://docs.travis-ci.com/user/environment-variables/#Encrypted-Variables)
+which works well for trusted branches and committers. For security reasons,
+encrypted variables are not exposed to untrusted pull requests, so third-party
+integrations do not work for pull requests.
+
+The JWT addon replaces encrypted variables with a time-limited authentication
+token, which is exposed to pull requests without security consequences.
+
+For this to work the JWT addon needs to be enabled in the `.travis.yml` file,
+and the third-party need to have integrated with the JWT service and allow
+token-based authentication.
+
+<img src="/user/images/travis_jwt.svg" alt="JWT Travis Flow Diagram">
+
+
+### .travis.yml
+
+Add the encrypted key to the `jwt` section of the `.travis.yml`
+file. For example:
+
+```yml
+addons:
+  jwt:
+     secure: <SAUCE_ACCESS_KEY ENCRYPTED>
+```
+
+You can also configure several services:
+
+```yml
```

Comment:
The data structure of `jwt` seems a bit strange. Looking at the code, the addon simply discards the keys given below. It seems more intuitive to me to have:

```yaml
addons:
  jwt:
    - secure: <SECRET_1>
    - secure: <SECRET_2>
```